mne/io/edf/edf.py

```diff
@@ -109,7 +109,7 @@ def _read_segment_file(self, data, idx, fi, start, stop, cals, mult):
         data_size = self._raw_extras[fi]['data_size']
         data_offset = self._raw_extras[fi]['data_offset']
         stim_channel = self._raw_extras[fi]['stim_channel']
-        tal_channel = self._raw_extras[fi]['tal_channel']
+        tal_channels = self._raw_extras[fi]['tal_channel']
```

Comment:
Not as input to `Raw` constructor, but I think this happens with `concatenate_raws`